opensrp-anc/src/test/java/org/smartregister/anc/library/interactor/RegisterInteractorTest.java

```diff
@@ -62,8 +67,12 @@
     @Captor
     private ArgumentCaptor<Long> longArgumentCaptor;
 
+    @Mock
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5624791)